test/clt-tests/core/test-skipping-plugin-loading.rec

```diff
@@ -0,0 +1,194 @@
+––– block: ../base/start-searchd –––
+––– input –––
+mysql -h0 -P9306 -e "SHOW BUDDY PLUGINS\G;" | grep "manticoresoftware/buddy-plugin-replace"
+––– output –––
+Package: manticoresoftware/buddy-plugin-replace
+––– input –––
+mysql -v -h0 -P9306 -e "SHOW TABLES; DROP TABLE IF EXISTS tbl; SHOW TABLES; CREATE TABLE tbl (id BIGINT, model TEXT, storage_capacity INTEGER, color string, release_year INTEGER, price FLOAT, discounted_price FLOAT, sold BOOL, date_added TIMESTAMP, product_codes MULTI, values MULTI64, additional_info JSON, vector float_vector knn_type='hnsw' knn_dims='4' hnsw_similarity='l2');"
+––– output –––
+--------------
+SHOW TABLES
+--------------
+--------------
+DROP TABLE IF EXISTS tbl
+--------------
+--------------
+SHOW TABLES
+--------------
+--------------
+CREATE TABLE tbl (id BIGINT, model TEXT, storage_capacity INTEGER, color string, release_year INTEGER, price FLOAT, discounted_price FLOAT, sold BOOL, date_added TIMESTAMP, product_codes MULTI, values MULTI64, additional_info JSON, vector float_vector knn_type='hnsw' knn_dims='4' hnsw_similarity='l2')
+--------------
+––– input –––
+chmod +x ./test/clt-tests/core/generate-1mln-records.sh
+––– output –––
+––– input –––
+total_iterations=20 ./test/clt-tests/core/generate-1mln-records.sh
+––– output –––
++-------+------+
+| Index | Type |
++-------+------+
+| tbl   | rt   |
++-------+------+
++----------+
+| count(*) |
++----------+
+|     1000 |
++----------+
+––– input –––
+mysql -h0 -P9306 -e "REPLACE INTO tbl SET model ='iPhone 15 PRO' WHERE id = 101; select * from tbl WHERE id = 101;"
```

Comment:
I believe using `\G` instead of `;` is much clearer for tests in this case to get rid of many dashes in the output.

Reply:
Done